curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-arrow-functions-to-write-concise-anonymous-functions.md

```diff
@@ -32,7 +32,7 @@ When there is no function body, and only a return value, arrow function syntax a
 const myFunc = () => "value";
 ```
 
-This code will still return <code>value</code> by default.
+This code will still return <code>"value"</code> by default.
```

Comment:
```suggestion
This code will still return the string <code>value</code> by default.
```

Reply:
@koehlertimo Can you commit @nhcarrigan's suggestion by click the `Commit suggestion` button if you are fine with this suggestion?  Thanks.